src/views/Projects/index.jsx

```diff
@@ -50,22 +73,32 @@ export default class Projects extends Component {
     return (
       <Fragment>
         <header className={classes.header}>
-          <Typography variant="display2" align="center">
+          <Typography variant="display2" align="center" color="primary">
             Codetribute
           </Typography>
-          <Typography variant="subheading" align="center">
-            Find your first code contribution with Mozilla
-          </Typography>
-        </header>
-        <main className={classes.container}>
-          <TextField
-            label="Search"
-            fullWidth
+          <div className={classes.flexContainer}>
```

Comment:
To center the Typograhpy element, you can add `justifyContent: 'center'` to `flexContainer`.

```js
flexContainer: {
  display: 'flex',
  justifyContent: 'center',
},
```